web/askme/controllers/auth.go

```diff
@@ -63,7 +71,15 @@ func (c *AuthController) performLogin(cxt framework.Context) {
 }
 
 func (c *AuthController) signup(cxt framework.Context) {
-	c.Render(cxt.ResponseWriter(), framework.ViewModel{BodyTmpl: "signup.body", Title: "Signup", HeadTmpl: "signup.head", Bag: framework.Map{}})
+	c.Render(
+		cxt.ResponseWriter(),
+		framework.ViewModel{
+			BodyTmpl: "signup.body",
+			Title: "Signup",
+			HeadTmpl: "signup.head",
+			Bag: framework.Map{
+				"CSRF": cxt.Session().Get("csrf"),
```

Comment:
This might be challenging, but is there a better way to inject the CSRF key into the Bag if CSRF is enabled? so the developer doesn't need to explicitly do that?